DIP3E/Codes/Python/Chapter-3/Track/Gonzalez_example_3_1.py

```diff
@@ -0,0 +1,45 @@
+
+# coding: utf-8
+
+# In[4]:
+
+
+# Gozalez examples 3.1 
+# contrast enhance using gamma transformation 
+
+# importing matplotlib modules 
+import matplotlib.image as mpimg 
+import matplotlib.pyplot as plt
+
+# Read Images 
+img = mpimg.imread('Fig0308(a)(fractured_spine).tif') 
+  
+# Output Images 
+plt.imshow(img,'gray')
+
+
+# In[6]:
+
+
+img_b = img**0.6
+img_c = img**0.4
+img_d = img**0.3
+
+
+# In[8]:
+
+
+plt.imshow(img_b,'gray')
```

Comment:
Perhaps you can try to show all four images with one `plt.imshow`? Just like `Figure 3.8`